repos/system_upgrade/el7toel8/actors/checkmemcached/actor.py

```diff
@@ -0,0 +1,62 @@
+from leapp.actors import Actor
+from leapp.models import InstalledRedHatSignedRPM
+from leapp.reporting import Report
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+from leapp.libraries.common.reporting import report_generic
+from leapp.libraries.stdlib import CalledProcessError, run
+
+import re
+
+class CheckMemcached(Actor):
+    name = 'check_memcached'
+    description = 'Check for incompatible changes in memcached configuration.'
+    consumes = (InstalledRedHatSignedRPM,)
+    produces = (Report)
+    tags = (ChecksPhaseTag, IPUWorkflowTag)
+
+    def process(self):
+        installed_packages = set()
+
+        signed_rpms = self.consume(InstalledRedHatSignedRPM)
+        for rpm_pkgs in signed_rpms:
+            for pkg in rpm_pkgs.items:
+	        installed_packages.add(pkg.name)
```

Comment:
You have tab mixed in here with spaces. From pylint:

```
E: 23, 0: inconsistent use of tabs and spaces in indentation (<string>, line 23) (syntax-error)
```

Reply:
Should be fixed now.